containers/unit_tests/TestDualView.hpp

```diff
@@ -67,12 +67,26 @@ struct test_dualview_combinations {
   Scalar result;
 
   template <typename ViewType>
-  Scalar run_me(unsigned int n, unsigned int m) {
+  Scalar run_me(unsigned int n, unsigned int m, bool with_init) {
     if (n < 10) n = 10;
     if (m < 3) m = 3;
-    ViewType a("A", n, m);
 
-    Kokkos::deep_copy(a.d_view, 1);
+    ViewType a;
+
+    if (with_init) {
+      a = ViewType("A", n, m);
+      Kokkos::deep_copy(a.d_view, 1);
+    } else {
+      a = ViewType(Kokkos::ViewAllocateWithoutInitializing("A"), n, m);
+
+      Kokkos::parallel_for(
+          Kokkos::RangePolicy<execution_space>(0, n),
+          KOKKOS_LAMBDA(const int i) {
+            for (int j = 0; j < m; j++) {
+              a.d_view(i, j) = 1;
+            }
```

Comment:
Why a `parallel_for` here instead of `deep_copy`?

Reply:
Really just trying to emulate the typical use case.  Most times without initialization is selected is because the user code will turn around and launch a kernel that initializes the data to something other than the default, thus eliminating the extra kernel launch.

Reply:
That's literally what `deep_copy()` does :)